### PR TITLE
[dev-image] Add support for pre-commit

### PR DESCRIPTION
**Pull request by aledbf**
Example of validation failure:
Screenshot from 2021-06-08 13-47-49

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  rev: v4.0.0
+  hooks:
+    #- id: check-yaml
+    #  args: [--allow-multiple-documents]
+    - id: end-of-file-fixer
+    - id: trailing-whitespace
+    - id: check-symlinks
+    - id: mixed-line-ending
+
+- repo: https://github.com/jumanjihouse/pre-commit-hooks
```
Comment (csweichel) [inline .pre-commit-config.yaml:12]: We should probably fork/vendor this, just a precautionary measure (supply chain attacks or if the repo goes away).

**Review by csweichel — APPROVED** [review]
Other than forking the `jumanjihouse/pre-commit-hooks` repo. this lgtm

File: .pre-commit-config.yaml (hunk 1 above)
Comment (aledbf) [reply .pre-commit-config.yaml:12]: fixed

**Pull request merged**